src/pages/profiles/ProfileList.tsx

```diff
@@ -65,6 +65,10 @@ const ProfileList: FC = () => {
 
   const featuresProfiles = project?.config["features.profiles"] === "true";
 
+  profiles.sort((p1, p2) =>
+    p1.name === "default" ? -1 : p2.name === "default" ? 1 : 0
```

Comment:
Shall we define a lambda - like `defaultNameFirst` that we can import and reuse for the three sorts callss as argument?

Reply:
Sure - good idea. I will create it and add it to a helper util.